src/components/fileBrowser/FileTable.vue

```diff
@@ -1,17 +1,21 @@
 <template>
     <div class="flex flex-col mx-2">
         <div class="overflow-x-auto">
-            <div class="py-2 align-middle inline-block min-w-full">
-                <div class="shadow overflow-hidden border-b border-gray-200 sm:rounded-lg">
-                    <FileDropArea class="h-full" @send-file="uploadFiles">
+            <div class="py-2 px-4 align-middle inline-block min-w-full">
+                <div class="flex justify-end mb-2">
+                    <ViewSelect />
+                </div>
+                <div class="overflow-hidden border-b border-gray-200 sm:rounded-lg">
+                    <FileDropArea @click.stop class="h-full" @send-file="uploadFiles">
```

Comment:
Added a @click.stop to propagate the "deselect all files by clicking the gray background" when clicking around in the filebrowser itself